tests/conftest.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+# Constants for testing with extra plugin arguments
+_NO_ARGS = []
+_XDIST_ZERO = ["--numprocesses", "0"]
+_XDIST_TWO = ["--numprocesses", "2"]
+
+
+@pytest.fixture(
+    params=[_NO_ARGS, _XDIST_ZERO, _XDIST_TWO],
+    ids=["no_plugin", "xdist_zero", "xdist_two"],
+)
+def plugin_args(request: pytest.FixtureRequest) -> list[str]:
```

Comment:
```suggestion
def plugin_args(request: pytest.FixtureRequest) -> List[str]:
```

For py3.8 compatibility. (I'll start work on syrupy v5 soon to drop py3.8 support since it's EOL)

Reply:
Started some work to drop py3.8 in syrupy v5 but no timeline on that